docs/topics/validation-report.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/rules-development-guide/master.adoc
 
+:_content-type: REFERENCE
 [id="validation-report_{context}"]
```

Comment:
@stoobie @sbeskin-redhat Modules with titles that begin "About..." are generally concept modules. Although this one has some details, I don't think it's a reference module.

Reply:
@RichardHoch  But this title doesn't begin with "About"...

Reply:
@sbeskin-redhat  Line 6 reads: = About validation reports.  when I wrote "the title," I meant the Heading 1 text, not the file name. Sorry if that was not clear.

Reply:
@RichardHoch  Oh, I see. OK, I'll make it a concept.

Reply:
I have nothing to add.